frontend/src/components/DefaultInput/input.tsx

```diff
@@ -0,0 +1,16 @@
+import { InputHTMLAttributes } from "react";
+import * as S from "./styles";
+
+type InputProps = {
```

Comment:
Já que não tem nenhuma prop adicional à tipagem já definida, pode remover o `{} &` e só igualar.

```suggestion
type InputProps = InputHTMLAttributes<HTMLInputElement>
```